Source/Blazorise/Components/FilePicker/FilePicker.razor.cs

```diff
@@ -434,6 +440,38 @@ protected Task OnProgressed( FileProgressedEventArgs fileProgressedEventArgs )
         /// </summary>
         [Parameter] public bool DisableProgressReport { get; set; } = false;
 
+        /// <summary>
+        /// Gets or Sets whether file picker should upload directories.
+        /// </summary>
+        [Parameter]
+        public bool Directory
+        {
+            get
+            {
+                return directory;
+            }
+            set
+            {
+                if ( value )
```

Comment:
We should exit early if nothing changes.

eg.

```
if ( directory = value )
  return;
```

Reply:
OBE